modules/extensions/src/data-filter/data-filter-extension.ts

```diff
@@ -316,27 +316,30 @@ export default class DataFilterExtension extends LayerExtension<
       } = this.getAttributeManager()!;
       filterModel.setVertexCount(this.getNumInstances());
 
-      this.context.device.clearWebGL({framebuffer: filterFBO, color: [0, 0, 0, 0]});
-
-      filterModel.updateModuleSettings(params.moduleParameters);
```

Comment:
@felixpalmer Not sure how to deal with the `moduleParameters` they are completely untyped, not sure if they have the right structure for `shaderInputs.setProps()`

Reply:
Could you review https://github.com/visgl/deck.gl/pull/9158 ? I believe I've handled it there